Xamarin.Forms.Mocks/PlatformServices.cs

```diff
@@ -50,6 +52,20 @@ public string GetMD5Hash(string input)
             throw new NotImplementedException();
         }
 
+        public string GetHash(string input)
+        {
+            var stringBuilder = new StringBuilder();
+            using (var hash = SHA256.Create())
+            {
+                var enc = Encoding.UTF8;
+                byte[] result = hash.ComputeHash(enc.GetBytes(input));
+                foreach (byte b in result)
+                    stringBuilder.Append(b.ToString("x2"));
+            }
+
+            return stringBuilder.ToString();
+        }
```

Comment:
Ok, so it looks like `GetHash` is new in 4.7.x. A question, I don't know the answer to.

If we just add this method, but leave everything targeting Xamarin.Forms 4.6, will it fix 4.7.x?

You could  revert some of the files here, and CI will produce a new `.nupkg` you can test:

https://ci.appveyor.com/api/buildjobs/e3yvuauteo2bui9q/artifacts/Xamarin.Forms.Mocks.zip

Thanks!

Reply:
Unfortunately it didn't help: https://ci.appveyor.com/project/jonathanpeppers/xamarin-forms-mocks/builds/32642522/artifacts

Reply:
that method is not found in interface where so, we have to use XF 4.7